parcel.py

```diff
@@ -857,6 +870,16 @@ def _get_stages_for_parcel(parcel):
     flask.abort(400)
 
 
+@parcel_views.route('/pick_products', methods=['GET'])
+def pick_products():
+    if flask.request.method == "GET":
+        id_lot = flask.request.args.get('id', 'lot1')
+        delivery_type = flask.request.args.get('delivery_type', 'lot')
```

Comment:
Store these default values in two variables, `DEFAULT_LOT` and `DEFAULT_DELIVERY_TYPE` in `definitions.py`